src/main/java/com/google/devtools/build/lib/bazel/rules/genrule/BazelGenRuleRule.java

```diff
@@ -90,7 +90,7 @@ working directory. If run directly (using the <code>standalone</code> strategy),
   results, including caching policies and environment variables. Tests generally need to be run
   after the build is complete and on the target architecture, whereas genrules are executed during
   the build and on the exec architecture (the two may be different). If you need a general purpose
-  testing rule, use <a href="${link sh_test}"><code>sh_test</code></a>.
+  testing rule, use <a href="shell#sh_test"><code>sh_test</code></a>.
```

Comment:
I couldn't get `${link sh_test}` to work and dropped it since it only occurs a single time in the docs.